<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authorization</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgb(190, 152, 255);*/
/*background-color: rgb(0, 188, 212); лазурный*/ 
/*background-color: rgb(153,255,153); салатовый */
background-color: rgb(1, 200, 255);
color: rgb(79, 56, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,2,4,2" columnstretch="0,3,10,3,0">
    <item row="2" column="3">
     <widget class="ScaledPixmap" name="rightPicture">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>175</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>700</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="ScaledPixmap" name="labelPlane">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>250</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(220, 220, 220);
color: blue;
/*color:rgb(85, 0, 0);</string>
      </property>
      <property name="title">
       <string>Введите логин и пароль для входа</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,4">
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelLogin">
           <property name="styleSheet">
            <string notr="true">background:rgb(220, 220, 220);
color: blue;
/*color: rgb(79, 75, 255);</string>
           </property>
           <property name="text">
            <string>Логин:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineLogin">
           <property name="styleSheet">
            <string notr="true">background: white;
color: blue;

/*color: rgb(68, 0, 0); старый бордовый
color: black;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelPassword">
           <property name="styleSheet">
            <string notr="true">background:rgb(220, 220, 220);
color: blue;
/*
color: rgb(79, 75, 255);</string>
           </property>
           <property name="text">
            <string>Пароль:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="linePassword">
           <property name="styleSheet">
            <string notr="true">background: white;
color: blue;
/*
color: black;
color: rgb(68, 0, 0);*/</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true">color:rgb(0, 0, 127);
background: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(195, 195, 195, 255), stop:0.487562 rgba(255, 255, 255, 255), stop:1 rgba(195, 195, 195, 255))</string>
           </property>
           <property name="text">
            <string>Войти</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="ScaledPixmap" name="leftPicture">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>175</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>700</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QLabel" name="labelText">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>29</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
/*color: rgb(255, 255, 255); старый синий*/ 
</string>
      </property>
      <property name="text">
       <string>  Авторизация для администраторов турфирмы Bookit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaledPixmap</class>
   <extends>QLabel</extends>
   <header location="global">scaledpixmap.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
